apps/fyle/helpers.py

```diff
@@ -185,4 +186,32 @@ def sync_dimensions(fyle_credentials: FyleCredential, workspace_id: int) -> None
 
     platform.import_fyle_dimensions(import_taxes=True)
    
-    
+
+def construct_expense_filter(expense_filter:ExpenseFilter):
+    constructed_expense_filter = {}
+    if expense_filter.is_custom and expense_filter.operator != 'isnull':
+        filter1 = {
+            'custom_properties__{0}__{1}'.format(expense_filter.condition, expense_filter.operator): expense_filter.values if len(expense_filter.values) > 1 else expense_filter.values[0]
+        }
+        constructed_expense_filter = Q(**filter1)
+
+    elif expense_filter.is_custom and expense_filter.operator == 'isnull':
+        expense_filter_value: bool = True if expense_filter.values[0].lower() == 'true' else False
+        filter1 = {
+            'custom_properties__{0}__{1}'.format(expense_filter.condition, expense_filter.operator): expense_filter_value
+        }
+        filter2 = {
+            'custom_properties__{0}__exact'.format(expense_filter.condition): None
+        }
+        if expense_filter_value == True:
+            constructed_expense_filter = Q(**filter1) | Q(**filter2)
+        else:
+            constructed_expense_filter = ~Q(**filter2)
+
+    else:
+        filter1 = {
+            '{0}__{1}'.format(expense_filter.condition, expense_filter.operator):expense_filter.values if len(expense_filter.values) > 1 else expense_filter.values[0]
```

Comment:
Line too long, need to shorten it
```suggestion
            '{0}__{1}'.format(
                expense_filter.condition, 
                expense_filter.operator
            ):expense_filter.values if len(expense_filter.values) > 1 else expense_filter.values[0]
```